torch/ao/quantization/utils.py

```diff
@@ -547,6 +547,89 @@ def _patched_module_call(self, *args, **kwargs):
     return fqn_to_example_inputs
 
 
+def _get_fixed_params_observed_lstm(
+    float_lstm: torch.nn.LSTM,
+    is_qat: bool,
+    linear_output_qparams: Optional[Tuple[float, int]] = None,
```

Comment:
can we ask the user to pass in an observer/fake_quant constructor instead, so it works the same way as other APIs?

```
obs_ctr = FakeQuantize.with_args(...)
_get_lstm_with_individually_observed_parts(
    float_lstm,
    linear_obs_ctr,
    ...
)
```